07.ls_object/ls_directory.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative 'ls_file'
+
+class LsDirectory
+  def initialize(directory_path:, hidden_file_presence: true, reversed_order: false)
```

Comment:
なんでもかんでもキーワード引数にするとコードが冗長になりがちなので、指定が必須な引数は普通の引数にした方がいいかもしれません。

```suggestion
  def initialize(directory_path, hidden_file_presence: true, reversed_order: false)
```